common/src/test/java/com/microsoft/identity/common/SharedPreferencesAccountCredentialCacheTest.java

```diff
@@ -101,7 +101,8 @@ public void setUp() throws Exception {
         mSharedPreferencesFileManager = SharedPreferencesFileManager.getSharedPreferences(
                 testContext,
                 sAccountCredentialSharedPreferences,
-                new StorageHelper(testContext) // Use encrypted storage for tests...
+                -1,
```

Comment:
Same here - this looks to be paving over some changes I did in:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1444/files#diff-d8fd7f57e0a3e84b81cf6959b128979754729ab74636f5b67f4d5fa0d7fb7b2d